src/routes/games/+page.svelte

```diff
@@ -1,79 +1,81 @@
+        {#each games as game}
+            <li>
+                <a
+                    href={game.url}
+                    class="game flex flex-col items-start justify-start rounded-lg bg-white p-4 shadow-md hover:underline"
+                    style:--tag={game.viewTransitionTag}
+                >
+                    <img
+                        src={game.boxArtUrl}
+                        alt="Box art"
+                        class="mb-4 h-auto w-full rounded-t-lg"
+                    />
 
-        <p class="mt-2 leading-tight" />
-    </header>
-
-    <main class="self-stretch">
-        <ol class="grid grid-cols-1 gap-12 sm:grid-cols-2 md:grid-cols-3">
-            {#each games as game}
-                <li>
-                    <a
-                        href={game.url}
-                        class="game flex h-full w-full flex-col items-center justify-center hover:underline"
-                        style:--tag={game.viewTransitionTag}
-                    >
-                        <!-- TODO use picture tag -->
-                        <img
-                            src={game.boxArtUrl}
-                            alt="Box art"
-                            class="max-h-60"
-                        />
-
-                        <div class="mt-6 text-center leading-tight">
-                            <h2 class="text-2xl leading-5">{game.name}</h2>
-                            <div class="mt-2 text-sm opacity-80">
-                                {game.console}
-                            </div>
-                            <div class="text-sm opacity-80">
-                                {game.year}
-                            </div>
-                        </div>
-                    </a>
-                </li>
-            {/each}
-        </ol>
-    </main>
-
-    <p class="text-sm text-gray-400">
-        Box art courtesy of <a
-            href="https://gamesdb.launchbox-app.com/"
-            class="hover:underline">LaunchBox Games Database</a
-        >
-    </p>
-</div>
+                    <div class="w-full">
+                        <h2 class="text-2xl font-semibold text-gray-800">
+                            {game.name}
+                        </h2>
+                        <p class="mt-1 text-gray-500">
+                            {game.console}
+                        </p>
+                        <p class="text-sm text-gray-400">
+                            {game.year}
+                        </p>
+                    </div>
+                </a>
+            </li>
+        {/each}
+    </ol>
+</main>
 
 <style>
+    .w-full {
+        padding: 30px;
+    }
+
     .game {
-        position: relative;
+        display: flex;
+        flex-direction: column;
+        justify-content: start;
+        align-items: start;
+        padding: 4px;
+        background-color: white;
+        border-radius: 15px;
+        box-shadow: 0 2px 5px rgba(0, 0, 0, 0.15);
+        text-decoration: none;
```

Comment:
These can and should all be tailwind

`flex flex-col justify-start items-start p-2 bg-white rounded-md shadow no-underline`